### ISSUE TITLE
Radio Player Update

### ISSUE THREAD

**Issue by caprenter** [caprenter/bcb-website#116]
Radioplayer News
The New Radioplayer Web Player has arrived
 
The waiting is over

Radioplayer is delighted to announce the launch of the new browser based Web Player. The player - first launched in 2011 then updated in 2014 and 2017 - has been redesigned and rewritten to ensure it works well for modern browsers. And we've added some great new features.
What's new?

· Refreshed contemporary look which works well on computers and mobile devices alike

· Widgets which show your now playing information, schedule and podcasts if you're sending these to us

· Support for next-gen streaming formats like HLS and MPEG Dash

· Updates to monetisation options including AdsWizz, Triton and VAST support

· Lots of tweaks behind-the-scenes to improve reliability and flexibility
 
It's time to upgrade

To ensure a smooth transition for listeners, we're asking all stations to upgrade to the new v4 web player. Because it's important for listeners to have the same experience across all UK stations, this is a mandatory upgrade and you have until 4th September to update your player. But don't wait this long. Upgrade today.
Upgrading is free and easy

The new web console generator in your Station Control Panel account has everything you need to upgrade. Streams can be managed from the Audio tab in your station profile, which means if you change streams in the future, you won't need to generate a new player again.
We've got lots of docs

Start by checking out What's New and when you're ready you can follow these steps to generate a v4 player. Remember you need to go through the generation process for each station - you can't clone web player for multiple stations otherwise things like favourites will break and not look right, and your stats will be wrong.
Tell us once you've upgraded

The new web player offers lots of customisation options. We'd like to see what you come up with - and give it a quick check. Once you've upgraded your web player drop us a line at station.support@radioplayer.co.uk. Remember to include the web address of your new player.
If you want to use widgets but aren't sending us metadata

Now is the time to start sending us Now Playing data, podcasts and schedule information. Check out our knowledgebase for specific advice about how to do this.
Stuck? We can help!

As well as our knowledgebase, don't forget you can always ask us questions by emailing station.support@radioplayer.co.uk or tweeting your question to us @RPStationHelp

**Comment by caprenter** [comment]
I've logged into the console - we have login information documented. 
i've regenerated a v4 player and uploaded it so that you can see it:
https://www.bcbradio.co.uk/console/

There are a couple of issues with it in terms of displaying the current show, and the 'logo' in the player section.

I think this is about the data we are sending about the shows. It may be that we don't send anything until after it is broadcast - hence the podcast section showing up, but not the live data.

**Comment by caprenter** [comment]
See also #117 as the non-secure items being served over https probably also applies here.

**Comment by caprenter** [comment]
@notshi We may also want to theme the new player. I've not looked into how it works that much.

**Comment by notshi** [comment]
Does this mean the new radio popup will be a full-width page?

**Comment by caprenter** [comment]
I don't think it needs to be full - width. If you have a play at resizing it, you will see how the columns work. We have the possibility of showing a third column of autogenertaed data if we have the data (which we don't yet) I think it does not show our latest tweets tho, like the existing player.

**Comment by caprenter** [comment]
I've switched over to the new player at https://www.bcbradio.co.uk/player/index.html

to integrate it into a page #146 we'd need add header information and so on. It's a bit of a task, but shouldn't be too much work really...probably needs a new page template or something

**Comment by notshi** [comment]
Just noticed the new player today for some reason(?), looks great!

It doesn't seem to be working right now for some reason, tho it could just be a hiccup.
The links under the ON DEMAND section works fine.
